public/content/lessons/2016/cross-products-extended/index.mdx

```diff
@@ -1,10 +1,256 @@
+/>
+
+If you choose the appropriate direction for that vector, the cases where the dot product is negative will be the same as those when the right-hand-rule orientation of $\vec{\mathbf{u}}$, $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$ is negative. For example, a quick way to reverse the orientation is to swap $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$ which inverts the volume of the parallelepided.
+
+<Figure
+    image="./figures/the-idea/CrossProductParallelogramAndDotProductNegative.svg" 
+/>
+
+This means we just found a vector $\vec{\mathbf{p}}$ so that taking a dot product between $\vec{\mathbf{p}}$ and some vector $\vec{\mathbf{u}}$ is the same thing as computing the determinant of a 3x3 matrix whose columns contain the coordinates of that vector, $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$! 
+
+<Figure
+    image="./figures/the-idea/DeterminantAndDotProduct.svg" 
+/>
+
+So the answer we found computationally earlier using the notational trick must correspond to this vector. This is the fundamental reason why the computation and the geometric interpretation are related!
+
+## Conclusion
+
+To sum up what just happened, we defined a linear transformation from 3d space to the number line defined in terms of the vectors $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$.
+
+<Figure
+    image="./figures/conclusion/3dTo1dLinearTransformation.svg" 
+    caption="3d to 1d Linear Transformation"
+/>
+
+Then we went through two different ways to think about the dual vector of this transformation, the vector such that applying the transformation is the same as taking a dot product with this vector.
+
+<Figure
+    image="./figures/conclusion/DualVectorOfTransformation.svg" 
+    caption="Dual Vector of Transformation"
+/>
+
+On the one hand, thinking about the dual vector from a computational approach leads us to the trick of plugging in the symbols $\hat{\imath}$, $\hat{\jmath}$, and $\hat{k}$ to the first column of a matrix and computing the determinant.
+
+<Figure
+    image="./figures/conclusion/DualVectorComputationalApproach.svg" 
+/>
+
+But thinking geometrically, we can deduce that this dual vector must be perpendicular to $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, with a length equal to the area of the parallelogram spanned out by those two vectors. Also, it's direction will be determined by the right-hand rule.
+
+<Figure
+    image="./figures/conclusion/DualVectorGeometricApproach.svg" 
+/>
+
+For both approaches, finding the dual vector of this linear transformation gives us a much deeper understanding of the result of the cross product between two vectors $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$ than just the formula.
+
+<Figure
+    image="./figures/conclusion/CrossProductFormula.svg" 
+/>
+
+Next up are two important concepts in linear algebra: <LessonLink id="cramers-rule">Cramer's Rule</LessonLink> and <LessonLink id="change-of-basis">Change of basis</LessonLink>.
+
+## Practice
```

Comment:
Does it make sense to try and get the student to think along the lines of both the computational and geometric approach using practice questions in this lesson? I struggled to think of a natural place to inject a question into “the idea” section, but perhaps questions at the end could serve to test the students comprehension of both the notational trick (by gathering like expressions) or finding the signed volume of a parallelepiped and then finding the dual vector associated with the transformation function that related to the parallelepiped. Or, alternatively, is it enough to provide some questions about the cross-product of two vectors, and leave it up to the student to try and apply one or both approaches?

Reply:
That's a good question. One possible comprehension question would be to give the coordinates of three vectors, v, w, u, and ask for v dot (w cross u), and also (v cross w) dot u, and to mention in the explanation how if one understands the meaning here, you could find it by calculating the 3x3 determinant of a matrix holding all their vectors.